extension-manifest-v2/test/src/Dispatcher.test.js

```diff
@@ -23,7 +23,7 @@ describe('dispatcher tests', () => {
 
 	test('dispatcher is triggered once for conf value not in SYNC_ARRAY', () => {
 		conf.paused_blocking = true;
-		return expect(spy).toHaveBeenCalledTimes(1);
+		return expect(spy).toHaveBeenCalledTimes(2);
```

Comment:
Not a problem, but curious why it is now called twice? From the code change, I wouldn't had expected because it appears to move the "metrics.setUninstallUrl" up.

Reply:
comes from here https://github.com/ghostery/ghostery-extension/pull/946/files#diff-086936c62c02dba1767dca312056acb75a897c13f613da59bb2ff68940df79ffR75 
but this change has no sideeffects as no one else is listening to this event